SetupDataPkg/Tools/ConfigEditor.py

```diff
@@ -784,6 +850,34 @@ def load_from_delta(self):
             return
         self.load_delta_file(path)
 
+    def set_variable_runtime(self):
+        self.update_config_data_on_page()
+        if (not ask_yes_no("Do you want to save the variable to the system?\n")):
+            return
+
+        runtime_var_delta_path = "RuntimeVarToWrite.vl"
+        bin = b''
+        for idx in self.cfg_data_list:
+            if self.cfg_data_list[idx].config_type == 'xml':
+                bin = self.cfg_data_list[idx].cfg_data_obj.generate_binary_array(True)
+
+        with open(runtime_var_delta_path, "wb") as fd:
+            fd.write(bin)
+
+        uefi_var_write.set_variable_from_file(runtime_var_delta_path)
+        self.load_variable_runtime()
+        self.output_current_status("Settings are set to system and save to RuntimeVar.vl")
+
+    def del_all_variable_runtime(self):
+        if (not ask_yes_no(f"Do you want to delete all variables in {self.config_xml_path} on system?\n")):
+            return
+
+        schema = Schema.load(self.config_xml_path)
+        for knob in schema.knobs:
+            self.output_current_status(f"Delete variable {knob.name} with namespace {knob.namespace}")
+            uefi_var_write.delete_var_by_guid_name(knob.name, knob.namespace)

```

Comment:
```suggestion
            rc = uefi_var_write.delete_var_by_guid_name(knob.name, knob.namespace)
            if rc == 0:
                self.output_current_status(f"{knob.name} variable was not deleted from system {rc}")
            else:
                self.output_current_status(f"{knob.name} variable is deleted from system")```

Reply:
we want to print the output results accurately, if the variables dont' exist, we want that reflected in the logs.

Reply:
addressed in 
https://github.com/microsoft/mu_feature_config/pull/413/commits/e6f697b5cb45f996a49bfe3faaf16de4b177f7f0